test/functionalTest/cases/3008_get_registration/get_reg_using_v2_created_using_v1.test

```diff
@@ -0,0 +1,148 @@
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET registration using APIv2, created using V1
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# At the moment of implementing the functionality of issue #3008, there was no way to create
+# registrations using APIv2 (Nov/Dec 2017), so, to test this new functionality of querying registrations
+# in APIv2, the registrations had to be created using APIv1.
+# Now, one might say, so, as soon as APIv2 implements creation of registrations, this test should be changed to
+# use APIv2 registration creation instead.
+# Sure, but, creating and getting registrations in pure APIv2 will be sufficiently tested in the future.
+# I would prefer to keep this tests as is, creation in APIv1 and getting in APIv2, to test the 'API version cross'.
+#
+# 01. Register entity E1/A1, using APIv1
+# 02. GET the registration using APIv2
+#
+
+echo "01. Register entity E1/A1, using APIv1"
+echo "======================================"
+payload='{
+  "contextRegistrations": [
+  {
+    "entities": [
+      {
+         "id": "E1",
+         "isPattern": "false",
+         "type": "T1"
+      },
+      {
+         "id": "E2",
+         "type": "T2"
+      }
+    ],
+    "attributes": [
+      {
+        "name": "A1",
+        "type": "string",
+        "isDomain": "false"
+      },
+      {
+        "name": "A2",
+        "type": "string",
+        "isDomain": "false"
+      }
+    ],
+    "providingApplication": "http://localhost:'${CP1_PORT}'/v2"
+    }
+ ],
+ "duration": "P1M"
+}'
+orionCurl --url /v1/registry/registerContext --payload "$payload"
+REG_ID=$(echo "$_response" | grep registrationId | awk -F\" '{ print $4}')
+echo
+echo
+
+
+echo "02. GET the registration using APIv2"
+echo "===================================="
+orionCurl --url /v2/registrations/${REG_ID} -X GET
+echo
+echo
+
+
+--REGEXPECT--
+01. Register entity E1/A1, using APIv1
+======================================
+HTTP/1.1 200 OK
+Content-Length: 74
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "duration": "P1M",
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+02. GET the registration using APIv2
+====================================
+HTTP/1.1 200 OK
+Content-Length: 301
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "dataProvided": {
+        "attrs": [
+            "A1",
+            "A2"
+        ],
+        "entities": [
+            {
+                "id": "E1",
+                "type": "T1"
+            },
+            {
+                "id": "E2",
+                "type": "T2"
+            }
+        ]
+    },
+    "forwardingInformation": {
+        "timesSent": 0
```

Comment:
Maybe you could add a test with a CPr and cause some forwarding in order to see how this timesSent increases. Makes sense?

Reply:
...or maybe the forwarding logic has not been yet modified to increase timesSent each time a new query/update is forwarded?

Reply:
I haven't lifted a finger yet about forwards,
This PR only deals with GETting registrations.
Once the V2 forwards are implemented, we'll implement this kind of tests

NTC (for this particualar PR)

Reply:
For this particular PR I'd suggest not rendering the `forwardingInformation` element. If we leave as it is now, we are adding wrong information, i.e. we always say `"timesSent": 0` even when some forwareded request has been done. Better say nothing than a lie, I guess.

Reply:
Fixed in bd588f384